### PR TITLE
Add org.jabref.jabref

### PR DESCRIPTION
**Pull request by LyzardKing**
with fallback-x11 it won't work on wayland.
But with x11 it does. I'm not sure if it's running with xwayland or native wayland.

@koppor

### CODE REVIEW

**Comment by A6GibKm** [comment]
> I'm not sure if it's running with xwayland or native wayland.

You can test this with `xeyes`. The eyes won't follow the cursor when it is over a wayland app.

**Comment by LyzardKing** [comment]
Then it's running on x11

**Comment by LyzardKing** [comment]
If I try to run with wayland I get this error:
`Exception in thread "main" java.lang.UnsupportedOperationException: Unable to open DISPLAY`

**Comment by barthalion** [comment]
There's no metainfo file, that's why CI failed.

**Comment by LyzardKing** [comment]
@barthalion How can I find these errors (the metainfo one for instance) while building locally?
If I simply build the flatpak it works.
Is there a tool that I can use to determine what is needed by the flatpak?

**Comment by LyzardKing** [comment]
bot, build org.jabref.jabref

**Comment by LyzardKing** [comment]
bot, build org.jabref.jabref

**Comment by LyzardKing** [comment]
bot, build org.jabref.jabref

File: org.jabref.jabref.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    </p>
+  </description>
+  
+  <releases>
+        <release type="stable" version="5.0" date="2020-07-30T00:00:00Z">
+        </release>
+  </releases>
+  
+  <content_rating type="oars-1.0">
+  </content_rating>
```
Comment (bilelmoussaoui) [inline org.jabref.jabref.appdata.xml:23]:
```suggestion
  <content_rating type="oars-1.1" />
```

File: org.jabref.jabref.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (bilelmoussaoui) [inline org.jabref.jabref.appdata.xml:1]: This file should be sent upstream as well if possible.

**Comment by LyzardKing** [comment]
bot, build org.jabref.jabref

**Comment by LyzardKing** [comment]
The icon in the program is 64x64, not 128.
Is it possible to force the icon to be accepted

**Comment by bilelmoussaoui** [comment]
You can upscale the icon using something like imagemagick and include it in tree or do that at build time.

**Comment by LyzardKing** [comment]
bot, build org.jabref.jabref

**Comment by LyzardKing** [comment]
bot, build org.jabref.jabref

**Comment by LyzardKing** [comment]
how do I include imagemagick as a build dependency in flatpak?

**Comment by A6GibKm** [comment]
Does [this](https://github.com/flathub/org.bitbucket.mystery_keeper.VapourSynthEditor/blob/4bc60ef524aef96cf17fc489fb7f6e45e9243e1a/com.vapoursynth.VapourSynth.yaml) helps?

**Comment by nedrichards** [comment]
Yeah, there's a few options available https://github.com/search?q=org%3Aflathub+imagemagick&type=Code but basically, build it first then if you don't need it for runtime put a cleanup section in to remove it all from the final flatpak.

**Comment by LyzardKing** [comment]
@nedrichards Yes, that's what I did in the end...
It seems to work now, I believe.

**Pull request closed without merging**